pkg/network/protocols/http/ebpf_main.go

```diff
@@ -192,12 +193,13 @@ func newEBPFProgram(c *config.Config, offsets []manager.ConstantEditor, sockFD *
 	}
 
 	program := &ebpfProgram{
-		Manager:         errtelemetry.NewManager(mgr, bpfTelemetry),
-		cfg:             c,
-		offsets:         offsets,
-		subprograms:     subprograms,
-		probesResolvers: subprogramProbesResolvers,
-		tailCallRouter:  tailCalls,
+		Manager:           errtelemetry.NewManager(mgr, bpfTelemetry),
+		cfg:               c,
+		offsets:           offsets,
+		subprograms:       subprograms,
+		probesResolvers:   subprogramProbesResolvers,
+		tailCallRouter:    tailCalls,
+		classificationMap: classificationMap,
```

Comment:
We should have only one name, shall we rename `classificationMap` to `connectionMap` or rename `probes.ConnectionProtocolMap`